{
  "error": "unable to find connection by name 'this_is_wrong'"
}
event: id_assign
data: 2


